src/remote/inet.cpp

```diff
@@ -955,8 +955,7 @@ rem_port* INET_connect(const TEXT* name,
 	// Prepare hints
 	const bool ipv6 = os_utils::isIPv6supported();
 
-	struct addrinfo gai_hints;
-	memset(&gai_hints, 0, sizeof(gai_hints));
+	struct addrinfo gai_hints {};
```

Comment:
These `struct` are required only in C.

Reply:
What do you mean with this? 

Using `struct addrinfo gai_hints {};` will zero-initialize, just like the original call to memset did, but just with C++ idioms.

Reply:
```suggestion
	addrinfo gai_hints {};
```

Reply:
Ah, ok.

Reply:
Do you want me to remove all occurrences of such constructs in inet.cpp?

Reply:
Looks like a good thing to do at least in the lines/blocks you are changing.

Reply:
Done (for the lines I already touched).